schema.hh

```diff
@@ -909,6 +909,15 @@ public:
     // Search for an existing index with same kind and options.
     std::optional<index_metadata> find_index_noname(const index_metadata& target) const;
     friend std::ostream& operator<<(std::ostream& os, const schema& s);
+    /*!
+     * \brief print the CQL DESCRIBE output.
+     *
+     * CQL DESCRIBE is implemented at the driver level. This method mimic that functionality
+     * inside Scylla.
+     *
+     * It should be possible to use the method result to re-create the schema.
```

Comment:
Please be more explicit about what this function does schemas which **have** a view or an index, and for schemas which **are** a view or index itself. For example something like:
This function returns a "CREATE TABLE", "CREATE MATERIALIZED VIEW" or "CREATE INDEX" command depending on whether the given schema is a base table, a materialized view or a materialized view created for a secondary index. If the given schema is a base table, only the command to create the table itself is returned - its materialized views or indexes, if any, need to be created separately.

Reply:
ok